.eslintrc.js

```diff
@@ -72,6 +72,7 @@ module.exports = {
                 'error',
                 { 'ignoreDeclarationMerge': true },
             ],
+            'dot-notation': ['warn'],
```

Comment:
what's this one ?

Reply:
a['foo'] vs. a.foo

Reply:
ah voilà, donc celui ci va nous envoyer les fameuuuuses 46 warning, noice, tu peux résoudre.